docs/contributing/testing-guidelines/integration-testing.md

```diff
@@ -0,0 +1,293 @@
+# Integration Testing
+
+## Introduction
+
+This article motivates developers to adopt integration testing by explaining how to write, run, and evaluate the results of integration tests.
```

Comment:
Let's read and follow the Google developer documentation style guide as much as possible. :+1: 
https://developers.google.com/style/translation?hl=en#write-short,-clear,-and-precise-sentences

In this case, I think we can write like this:
```suggestion
This article explains how to write integration tests.
```

Reply:
See also: https://github.com/autowarefoundation/autoware/pull/14